test/README.md

```diff
@@ -0,0 +1,19 @@
+### How to run the tests?
```

Comment:
I think the majority of these details are already in TESTING.md. I've just added a link to TESTING.md on the README.